app/src/main/java/com/example/poste/activities/OptionsActivity.java

```diff
@@ -75,7 +63,5 @@ public void onComplete(Task<Void> task) {
                 startActivity(new Intent(OptionsActivity.this, LoginActivity.class));
```

Comment:
it would be beneficial to create a quick method that actually logs the user out instead of just moving them to login page; otherwise, simply pressing 'back' would be sufficient to steal the user session.

A User.logout() method would simply set all user fields to null (the token is particularly important)

Reply:
I agree with this. 
Additionally, might be slightly overkill, if an API call was made to the server, the server could invalidate the token it had generated for the user, making the application more secure as lingering tokens are a big security issue in most web applications.

Reply:
100%, and not overkill

Reply:
Created ne US for this:
https://tree.taiga.io/project/dzemlin-poste/us/221?no-milestone=1